### ISSUE TITLE
Postgres doesn't work

### ISSUE THREAD

**Issue by marblepebble** [docker-library/owncloud#9]
The Dockerfile needs pdo_pgsql added to the docker-php-ext-install line (pgsql alone isn't enough - it failed to connect to the db with an 'Unable to find driver' error without it).

Presumably the same is also true for mysql/pdo_mysql, but I didn't try that, not having a mysql instance handy.

**Comment by dowdell** [comment]
MySQL suffers from the same issue

**Comment by tianon** [comment]
Oh, both modules are required?  Can you tell us which version were you testing and what were the steps to reproduce so we can make sure our fix is complete?

**Comment by marblepebble** [comment]
Yes, both (well, all four: mysql, pdo_mysql, pgsql and pdo_pgsql) are required.
I set up a postgres container (postgres:latest, port 5432 mapped to port 5432), and connected from the host (which has the pgsql client installed):
psql -hlocalhost -Upostgres
CREATE USER owncloud WITH PASSWORD 'password';
CREATE DATABASE owncloud TEMPLATE template0 ENCODING 'UNICODE';
ALTER DATABASE owncloud OWNER TO owncloud;
GRANT ALL PRIVILEGES ON DATABASE owncloud TO owncloud;
\q
(as described in https://doc.owncloud.org/server/8.1/admin_manual/configuration_database/linux_database_configuration.html)
I then pulled owncloud:latest (8.1.0 at the time, linked it with the postgres container under the name 'postgres') and connected to it with my browser. I entered a username and password, opened up the database options part and selected postgres. I entered owncloud as the user, and the password I'd selected, owncloud as the database and postgres as the server name.
I then get an error "Could not find driver". I did a docker exec -it containername /bin/bash and docker-php-ext-install pdo_pgsql, and tried again and it worked.
